### DISCUSSION TITLE
How can I add information to every merchant?

### DISCUSSION THREAD

**Discussion by Xallistus** [datafeedr/wordpress-plugins#27] [Help]
Hi. I'm trying to build a price and feature comparison website for a certain niche. To stand out from the other comparison websites, I want to emphasize on the extra services the merchants maybe offering. For example: free delivery, the option to install the new device and the option to recycle the old device.

What would be the best way to add this information to every selected merchant?

Thank you in advance! :)

My sketch of what I'm trying to achieve:
![image](https://user-images.githubusercontent.com/100313670/204252767-8175fa00-4b9a-43a6-8bf9-fb1e5692e0e8.png)

**Comment by EricBusch** [comment]
The best way would be to modify the Comparison Set template ([more info here](https://datafeedrapi.helpscoutdocs.com/article/203-override-default-comparison-set-template)) and then modify the HTML and design to fit your needs. You can use `$dfrcs_product['merchant_id']` to write conditionals based on the Merchant's ID.

Just be very careful with Amazon. They have specific types of data which MUST be shown alongside their links. See Amazon's disclaimer policy for more info. 

Hope this helps!

**Reply by Xallistus** [reply]
Hi Eric,

Thank you for the quick reply! This will definitely help us. I was looking for such a function. I will pass this along to our developer. Also thanks for the advise regarding Amazon.

Keep up the good work. I have been following Datafeedr for a while now. So far, I'm very impressed with the quality and the high level of commitment you guys have with Datafeedr. So thank you!

Thymo

**Comment by EricBusch** [comment]
Hi Thymo

You're welcome! Here's the Amazon policies https://affiliate-program.amazon.com/help/operating/policies

Thanks for following us!

If you have any other questions, please don't hesitate to ask.

Thanks,
Eric
